src/odemis/odemisd/main.py

```diff
@@ -328,71 +329,98 @@ def _terminate_all_alive(self):
         It also stops the containers, once no component is running in them.
```

Comment:
Please update the docstring. It shouldn't talk about parents, but "dependents". "Instantiated children" are now called "dependencies", and "Delegated children" are "children".